src/tests/multihost/docs/layout.rst

```diff
@@ -1,13 +1,56 @@
-layout
+Layout
 ======
-This doc provides the layout of SSSD pytest framework and test suites directory.
+* This doc provides layout of sssd pytest framework and test suites directory.
+
 
 module
-------
-* sssd.testlib
+-------
 
-  This is the main top directory under which there are subdirectories containing various shared functions required to write tests using pytest
+* sssd.testlib
+        * This is the main top directory under which there are subdirectories
+          containing various shared functions required to write tests using
+          pytest.
 
 * sssd.testlib/common
+        * This directory contains modules related to configuring different
+          services like 389-ds, kerberos, samba, Joining to AD, SSH Login,
+          creating posix uses on 389-ds, Creating kerberos users in kerberos
+          database.
+
+* sssd.testlib/common/ipa
+        * This directory contains shared functions related to ipa like setting
+          up chrony, adding hbac rules, mapping external groups to posix
+          groups.
+
+
+Test Directories
+----------------
+
+* sssd/src/tests/multihost
+        * This is the parent directory containing all the pytest test suites.
+
+* sssd/src/tests/multihost/ad
+        * This is the directory containing the test suites related to AD like
+          automount, cifs, sudo, id mapping etc which require single AD server.
+
+* sssd/src/tests/multihost/admultidomain
+        * This is the directory containing test suites for SSSD AD
+          Provider Multi-domain and multi-forest tests.
```

Comment:
I think it might be worth to add the requirements, how many DCs from who many domains, here as well and also for the other sub-directories.